.npmignore

```diff
@@ -25,12 +25,12 @@ report/test-assets/
 
 # Exclude the CLI smoketests but keep the smoketest runner that is used
 # by downstream projects (e.g. publisher ads).
-!cli/test/smokehouse/**
-cli/test/smokehouse/test-definitions/
+!/cli/test/smokehouse/**
+/cli/test/smokehouse/test-definitions/
 
-results/
-lantern-data/
-timings-data/
+/results/
```

Comment:
okay maybe not this one? i think its referring to core/test/results which is where sample_v2.json is?
dunno.

Reply:
I'll revert, but that `results` is under `test` which is ignored anway 🤷